as-http.js

```diff
@@ -64,35 +65,28 @@ TCurlAsHttp.prototype.send = function send() {
 
     function onSent(err, head, stream, body) {
         if (err) {
-            if (self.onResponse) {
```

Comment:
The reason for this was to use tcurl as a library. Need to make sure no one is depending on it.